packages/edit-post/src/components/manage-blocks-modal/style.scss

```diff
@@ -54,6 +54,7 @@
 	top: 0;
 	padding: $panel-padding 0;
 	background-color: $white;
+	z-index: z-index(".edit-post-manage-blocks-modal__category-title");
```

Comment:
How is this change related?

Reply:
The headings for the block manager modal use `position: sticky`, creating a "sticky container" parent. Using a `position: relative` element (the wrapper for the input to position the checkbox) within the same sticky parent container, the browser treats them as in the same stacking context: 

![Screen Recording 2019-08-05 at 03 17 PM](https://user-images.githubusercontent.com/5375500/62489214-2c6e9c00-b794-11e9-8b44-df739bb411d2.gif)

Adding a z-index to the header makes the sticky header remain on top:

![Screen Recording 2019-08-05 at 03 20 PM](https://user-images.githubusercontent.com/5375500/62489403-a868e400-b794-11e9-8e5b-47208b9866b8.gif)